ironfish-cli/src/commands/wallet/post.ts

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CurrencyUtils, RawTransaction, RawTransactionSerde, Transaction } from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class PostCommand extends IronfishCommand {
+  static summary = 'Post a raw transaction'
+
+  static description = `Use this command to post a raw transaction.
+   The output is a finalized posted transaction. The transaction is also added to the wallet, and sent out to the network.`
+
+  static examples = [
+    '$ ironfish wallet:post 618c098d8d008c9f78f6155947014901a019d9ec17160dc0f0d1bb1c764b29b4...',
+  ]
+
+  static flags = {
+    ...RemoteFlags,
+    account: Flags.string({
+      description: 'The account that created the raw transaction',
+      required: false,
+    }),
+    confirm: Flags.boolean({
+      default: false,
+      description: 'Confirm without asking',
+    }),
+    offline: Flags.boolean({
+      default: false,
+      description: 'Post transaction offline',
+    }),
+  }
+
+  static args = [
+    {
+      name: 'transaction',
+      required: true,
+      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
+      description: 'The raw transaction in hex encoding',
+    },
+  ]
+
+  async start(): Promise<void> {
+    const { flags, args } = await this.parse(PostCommand)
+    const transaction = args.transaction as string
+
+    const serialized = Buffer.from(transaction, 'hex')
+    const raw = RawTransactionSerde.deserialize(serialized)
+
+    const client = await this.sdk.connectRpc()
+
+    if (!flags.confirm) {
+      let account = flags.account
+      if (!flags.account) {
+        const response = await client.getDefaultAccount()
+
+        if (response.content.account) {
+          account = response.content.account.name
+        }
+      }
+
+      if (account === undefined) {
+        this.error('Can not find an account to confirm the transaction')
+      }
```

Comment:
Do we need to load the default account here? It looks like we will load the default account in the RPC if one isn't passed in.

If we do want to load the default account before we hit the RPC endpoint, should we do it even if `--confirm` is passed?

Reply:
the account is used in confirm function. Its not related to the RPC.
In the confirm function, it will print the account in the confirm message.